clang/include/clang/Basic/Attr.td

```diff
@@ -1023,6 +1023,26 @@ def SYCLKernel : InheritableAttr {
   let Documentation = [Undocumented];
 }
 
+def SYCLScope : Attr {
+  let Spellings = [GNU<"sycl_scope">];
```

Comment:
Are we not spelling these with the CXX11 spellings?  Why not?

Reply:
Currently this attribute is only added by the Front End, and is not supposed to be used by end users,  only SYCL library/API developers might. But it is not used now even in the library, so I think I'll just remove the spelling then. Can be added when/if it is needed in the SYCL library/API.